### PR TITLE
Revise/correct a few non-human animal infectious disease labels

### LINKED ISSUE #7709: Revise/correct a few non-human animal infectious disease labels
Revise the following:

1. Neurologic disease, non-human animal (inadvertent EC to Borna Disease)
2. Ensure that viral, fungal, parasitic and bacterial non-human animal diseases are classified under infectious disease, non-human animal using EC relations.

### PR DESCRIPTION
**Pull request by katiermullen**
Closes #7709

### CODE REVIEW

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
@katiermullen 
A few comments about adding the animal affected taxon in the Logical definition. 
Remember that if you add only "infectious diseases, non human animals' and some infectious agent NCBITaxon###", then you mean that it is for any non-human.

I don't understand why you had to add the "viral non-human disease" as an asserted parent. It is not incorrect to add it, but you should "get it for free". Maybe we need to update the import for this infectious agent.

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -474902,6 +474907,7 @@ synonym: "peritonitis, infectious, feline" EXACT [MESH:D016766]
 xref: MESH:D016766 {source="MONDO:equivalentTo"}
 xref: UMLS:C0085306 {source="MONDO:equivalentTo"}
 is_a: MONDO:0024912 {source="MESH:D016766"} ! cat disease
+is_a: MONDO:0700053 {source="https://orcid.org/0000-0002-5002-8648"} ! viral infectious disease, non-human animal
 intersection_of: MONDO:0700049 ! infectious disease, non-human animal
 intersection_of: MONDO:0100332 NCBITaxon:11135 ! disease has primary infectious agent Feline infectious peritonitis virus
 relationship: in_taxon NCBITaxon:9681 {source="PMID:9364228", source="https://orcid.org/0000-0002-4142-7153", source="https://orcid.org/0000-0002-5002-8648", source="https://www.merckvetmanual.com/"} ! Felidae
```
[the remark this answers is not in the document]
Comment (katiermullen) [reply src/ontology/mondo-edit.obo:474913]: done

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -474816,6 +474820,7 @@ synonym: "Feline immunodeficiency virus caused cat disease" EXACT [MONDO:pattern
 xref: MESH:D016181 {source="MONDO:equivalentTo"}
 xref: UMLS:C0079335 {source="MONDO:equivalentTo"}
 is_a: MONDO:0024912 {source="MESH:D016181", source="MONDO:Redundant"} ! cat disease
+is_a: MONDO:0700053 {source="https://orcid.org/0000-0002-5002-8648"} ! viral infectious disease, non-human animal
 intersection_of: MONDO:0700049 ! infectious disease, non-human animal
 intersection_of: MONDO:0100332 NCBITaxon:11673 ! disease has primary infectious agent Feline immunodeficiency virus
 relationship: in_taxon NCBITaxon:9681 {source="PMID:1382145", source="PMID:9364228", source="https://orcid.org/0000-0002-4142-7153", source="https://orcid.org/0000-0002-5002-8648"} ! Felidae
```
[the remark this answers is not in the document]
Comment (katiermullen) [reply src/ontology/mondo-edit.obo:474826]: done

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -113211,6 +113211,7 @@ xref: MESH:D016182 {source="MONDO:equivalentTo", source="EFO:0007542"}
 xref: UMLS:C0080323 {source="MONDO:equivalentTo"}
 is_a: MONDO:0005583 {source="EFO:0007542", source="MESH:D016182/inferred"} ! non-human animal disease
 is_a: MONDO:0024985 ! sheep disease
+is_a: MONDO:0700053 {source="https://orcid.org/0000-0002-5002-8648"} ! viral infectious disease, non-human animal
 intersection_of: MONDO:0700049 ! infectious disease, non-human animal
 intersection_of: MONDO:0100332 NCBITaxon:2169971 ! disease has primary infectious agent Visna-maedi virus
 relationship: in_taxon NCBITaxon:9940 {source="https://orcid.org/0000-0002-4142-7153", source="https://orcid.org/0000-0002-5002-8648", source="https://www.merckvetmanual.com/"} ! Ovis aries
```
[the remark this answers is not in the document]
Comment (katiermullen) [reply src/ontology/mondo-edit.obo:113217]: done

File: src/ontology/mondo-edit.obo [hunk 4]
```diff
@@ -474921,6 +474927,7 @@ synonym: "swine infertility and respiratory syndrome" EXACT [MESH:D019318]
 xref: MESH:D019318 {source="MONDO:equivalentTo"}
 xref: UMLS:C0376538 {source="MONDO:equivalentTo"}
 is_a: MONDO:0024990 {source="MESH:D019318"} ! swine disease
+is_a: MONDO:0700053 {source="https://orcid.org/0000-0002-5002-8648"} ! viral infectious disease, non-human animal
 intersection_of: MONDO:0700049 ! infectious disease, non-human animal
 intersection_of: MONDO:0100332 NCBITaxon:28344 ! disease has primary infectious agent Porcine reproductive and respiratory syndrome virus
```
[the remark this answers is not in the document]
Comment (katiermullen) [reply src/ontology/mondo-edit.obo:474932]: done

File: src/ontology/mondo-edit.obo [hunk 5]
```diff
@@ -474952,6 +474959,7 @@ xref: MESH:D053570 {source="MONDO:equivalentTo"}
 xref: UMLS:C1721016 {source="MONDO:equivalentTo"}
 is_a: MONDO:0005583 {source="EFO:1000048"} ! non-human animal disease
 is_a: MONDO:0024990 {source="MESH:D053570"} ! swine disease
+is_a: MONDO:0700053 {source="https://orcid.org/0000-0002-5002-8648"} ! viral infectious disease, non-human animal
 intersection_of: MONDO:0700049 ! infectious disease, non-human animal
 intersection_of: disease_has_infectious_agent NCBITaxon:85708 ! Porcine circovirus 2
 relationship: in_taxon NCBITaxon:9823 {source="https://orcid.org/0000-0002-4142-7153", source="https://orcid.org/0000-0002-5002-8648"} ! Sus scrofa
```
[the remark this answers is not in the document]
Comment (katiermullen) [reply src/ontology/mondo-edit.obo:474965]: done

File: src/ontology/mondo-edit.obo [hunk 6]
```diff
@@ -474446,6 +474447,7 @@ synonym: "Distempers" RELATED [MESH:D004216]
 synonym: "Distempers, canine" RELATED [MESH:D004216]
 xref: MESH:D004216 {source="MONDO:equivalentTo"}
 is_a: MONDO:0024919 {source="MESH:D004216"} ! dog disease
+is_a: MONDO:0700053 {source="https://orcid.org/0000-0002-5002-8648"} ! viral infectious disease, non-human animal
 intersection_of: MONDO:0700049 ! infectious disease, non-human animal
 intersection_of: MONDO:0100332 NCBITaxon:11232 ! disease has primary infectious agent
```
[the remark this answers is not in the document]
Comment (katiermullen) [reply src/ontology/mondo-edit.obo:474452]: I think this is an error. The EC should be 'infectious disease, non-human animal'
 and ('disease has primary infectious agent' some 'canine distemper virus'). Do I need to update the import to include 'canine distemper virus'? If so, can you please tell me how :) Thank you!

**Review by sabrinatoro — APPROVED** [review]
looks good to me. Good job, @katiermullen

**Pull request merged**